dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Csv/UserImportRow.cs

```diff
@@ -0,0 +1,28 @@
+using CsvHelper.Configuration.Attributes;
+
+namespace TeacherIdentity.AuthServer.Services.Csv;
+
+public class UserImportRow
+{
+    public const string IdHeader = "ID";
+    public const string EmailAddressHeader = "EMAIL_ADDRESS";
+    public const string FirstNameHeader = "FIRST_NAME";
+    public const string LastNameHeader = "LAST_NAME";
+    public const string DateOfBirthHeader = "DATE_OF_BIRTH";
+
+    [Name(IdHeader)]
```

Comment:
Do we need `[Index]` attributes if we have column names?

Reply:
I guess if all the columns are there then the order doesn't matter